Assignment/solution_student/index.html

```diff
@@ -5,27 +5,94 @@
     <title>Relaxr Blog</title>
     <meta name="description" content="An HTML Starter Template">
     
-    <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
-    <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,600,800' rel='stylesheet' type='text/css'>
 
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
+  
   <body>
 
-    <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
     
       <header>
-        <h1>FEWD HTML Template</h1>
+          <h1 class="logo">Relaxr</h1>
+          <nav>
+            <ul>
+              <li><a href="#">About</a></li>
+              <li><a href="#">FAQ</a></li>
+              <li><a href="#">Team</a></li>
+              <li><a href="#">Contact Us</a></li>
+              <li><a href="#">Blog</a></li>
+            </ul>
+          </nav>
       </header>
 
-      <main>
-        <p>Our HTML starter template!</p>
-      </main>
+<!--enjoy this delightful lorem ipsum-->
 
-      <footer>
-        <p>&copy; FEWD6 & Friends, INC.</p>
+      <section class="content-wrapper clearfix">
```

Comment:
Hey, you nailed the part of creating elements to use as containers or wrappers and the clearfix, and that was what I was really hoping to see, so great job! Just wanted to point out that you shouldn't really be using the `<section>` purely as a means to wrap other elements for the sake of styling, this is where a `<div>` would make the most sense. Here's a little article that helps explain why: http://html5doctor.com/avoiding-common-html5-mistakes/#section-wrapper